README.md

```diff
@@ -172,15 +172,15 @@ sudo apt install libicu-dev libtinfo-dev libgmp-dev
 sudo dnf install libicu-devel ncurses-devel
 ```
 
-#### Windows-specific pre-requirements (optional)
+#### Windows-specific pre-requirements
 
-In order to avoid problems with long paths on Windows you can do the following:
+In order to avoid problems with long paths on Windows you can do either one of the following:
 
-1. In the `Local Group Policy Editor`: `Local Computer Policy -> Computer Configuration -> Administrative Templates -> System -> Filesystem` set `Enable Win32 long paths` to `Enabled` (Works
-   only for Windows 10).
+1. Clone the `haskell-ide-engine` to the root of your logical drive (e.g. to
+   `C:\hie`). If this doesn't work try the second option.
 
-2. Clone the `haskell-ide-engine` to the root of your logical drive (e.g. to
-   `C:\hie`)
+2. In the `Local Group Policy Editor`: `Local Computer Policy -> Computer Configuration -> Administrative Templates -> System -> Filesystem` set `Enable Win32 long paths` to `Enabled` (Works
```

Comment:
It seems `gpedir.msc` is not present in windows 10 home edition (my version!) so you have to edit directly the registry to enable long paths. Maybe a link to a official microsoft page with instructions (using regedit or gpedit.msc) would be useful: https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file#enable-long-paths-in-windows-10-version-1607-and-later

Reply:
I readed more carefully the link above and [this blog post](https://social.msdn.microsoft.com/Forums/en-US/50ecad58-b8cc-411a-af50-08a67dde5fca/long-name-support-is-not-enabled-on-windows-10?forum=windowssdk) and it seems that you have to enable the option system wide *and* the application have to state in a manifest file that it wants to use the option.
As cabal/stack and ghc are cli applications compiled with ghc i dont know if they would have that option enabled. We should check that setting the option system wide is enoug to let those tools manage a project with a deep path.

Reply:
> It seems `gpedit.msc` is not present in windows 10 home edition (my version!) so you have to edit directly the registry to enable long paths. Maybe a link to a official microsoft page with instructions (using `regedit` or `gpedit.msc`) would be useful: https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file#enable-long-paths-in-windows-10-version-1607-and-later
> 
> Moreover i would note that you have to open `gpedit.msc` to edit policies.

Good find !

Reply:
> As cabal, stack and ghc are cli applications compiled with ghc i dont know if they will have that option enabled.

It seems that ghc itself dont use a manifest for windows and the long path issue is still opened so i'am afraid that it will not work :-(
See 
https://gitlab.haskell.org/ghc/ghc/issues/13373
https://gitlab.haskell.org/ghc/ghc/issues/13796

Reply:
> so i'am afraid that it will not work :-(

It worked on my system regardless without this manifest